### PR TITLE
parent's work schedule - 187263042

### PR DESCRIPTION
**Pull request by guiburato**
#### 🔗 Pivotal Tracker ticket
[187263042](https://www.pivotaltracker.com/story/show/187263042)

#### ✍️ Description
new screens: 

- activities-work-schedule-vary
- activities-next-work-schedule
- activities-job-weekly-schedule
- activities-job-hourly-schedule
- activities-work-commute-time

#### 📷 Design reference
[Figma](https://www.figma.com/file/1oRFY7p6tdGogBJk7Ugle8/IL-CCAP-Full-Flow-(WIP)?type=design&node-id=1292-18984&mode=design&t=FgrsIRNl3MOq9Lz7-0)

#### ✅ Completion tasks

- [ x ] Added relevant tests
- [ x ] Meets acceptance criteria

### CODE REVIEW

File: src/main/resources/templates/gcc/activities-job-hourly-schedule.html [hunk 1]
```diff
@@ -1,18 +1,35 @@
 <!DOCTYPE html>
 <html th:lang="${#locale.language}" xmlns:th="http://www.thymeleaf.org">
-<head th:replace="~{fragments/head :: head(title='TEMPLATE')}"></head>
+<head th:replace="~{fragments/head :: head(title=#{activities-job-hourly-schedule.title})}"></head>
```
Comment (sree-cfa) [inline src/main/resources/templates/gcc/activities-job-hourly-schedule.html:3]: This page is missing validations 
Screenshot 2024-04-04 at 9 29 55 AM

**Review by sree-cfa — CHANGES_REQUESTED** [review]
Needs time validation

**Comment by sree-cfa** [comment]
The `activities-add-job-list` page can also be deleted from the repo since that's grouped in with `activities-add-job`

File: src/main/resources/templates/gcc/activities-job-hourly-schedule.html (hunk 1 above)
Comment (guiburato) [reply src/main/resources/templates/gcc/activities-job-hourly-schedule.html:3]: Done. 👍

**Review by sree-cfa — CHANGES_REQUESTED** [review]
ok nvm, it looks like I can't get "approval" status back to an unset state - just requesting message properties fix

**Review by sree-cfa — APPROVED** [review]
lgtm!

**Pull request merged**